metadata-ingestion/src/datahub/ingestion/source/sql/teradata.py

```diff
@@ -447,6 +447,11 @@ class TeradataConfig(BaseTeradataConfig, BaseTimeWindowConfig):
 @capability(SourceCapability.LINEAGE_COARSE, "Optionally enabled via configuration")
 @capability(SourceCapability.LINEAGE_FINE, "Optionally enabled via configuration")
 @capability(SourceCapability.USAGE_STATS, "Optionally enabled via configuration")
+@capability(
```

Comment:
the capability annotations support inheritance, so I don't think these are actually necessary

Reply:
wondered why we had no annotation on SQLAlchemySource source. Some like PLATFORM_INSTANCE, DOMAINS, SCHEMA_METADATA, CONTAINERS, DELETION_DETECTION can probably move in there.